site/src/styles/antd-overrides.less

```diff
@@ -65,6 +65,12 @@
 @marketing-block-width: 532px;
 @marketing-block-margin: 100px;
 
+@import '../components/steps.module.less';
+@process-icon-color: @armeria-magenta;
+@finish-icon-color: @armeria-magenta;
+@finish-tail-color: @armeria-magenta;
+@steps-description-max-width: 105px;
+
```

Comment:
The color seems too intense to me :sweat_smile: How about using Antd's default colors?
```suggestion
```